mobile/mysterium/entrypoint.go

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mysterium
+
+import (
+	"path/filepath"
+
+	log "github.com/cihub/seelog"
+	"github.com/mitchellh/go-homedir"
+	openvpn_core "github.com/mysteriumnetwork/go-openvpn/openvpn/core"
+	"github.com/mysteriumnetwork/node/cmd"
+	"github.com/mysteriumnetwork/node/core/connection"
+	"github.com/mysteriumnetwork/node/core/node"
+	"github.com/mysteriumnetwork/node/identity"
+	service_noop "github.com/mysteriumnetwork/node/services/noop"
+)
+
+// NewNode function creates new Node
+func NewNode() {
+	var di cmd.Dependencies
+
+	userHomeDir, err := homedir.Dir()
+	if err != nil {
+		panic(err)
+	}
+	dataDir := filepath.Join(userHomeDir, ".mysterium")
+	currentDir := userHomeDir
+
+	err = di.Bootstrap(node.Options{
+		node.OptionsDirectory{
+			dataDir,
+			filepath.Join(dataDir, "keystore"),
+			// TODO Embbed all config file to released artifacts
+			filepath.Join(currentDir, "config"),
+			// TODO Where to save runtime data
+			currentDir,
+		},
+
+		"127.0.0.1",
+		4050,
+
+		openvpn_core.NodeOptions{},
```

Comment:
With keyed fields, we can skip setting filed by the zero value.

Reply:
Used for some fields